src/prisma/prisma.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@nestjs/common';
-import { PrismaClient } from '@prisma/client';
+import {PrismaClient } from '@prisma/client';
```

Comment:
add space in between { and PrismaClient

import { PrismaClient } from '@prisma/client';